lib/active_record_shards/model.rb

```diff
@@ -3,24 +3,14 @@
 module ActiveRecordShards
   module Model
     def not_sharded
-      if self != ActiveRecord::Base && self != base_class
-        raise "You should only call not_sharded on direct descendants of ActiveRecord::Base"
-      end
-      self.sharded = false
+      ActiveSupport::Deprecation.warn("Calling not_sharded is deprecated. "\
+                                      "Please ensure to still read from the "\
+                                      "account db slave after removing the "\
+                                      "call.")
     end
 
     def is_sharded? # rubocop:disable Naming/PredicateName
```

Comment:
Should we seize the moment and rename `.is_sharded?` to `.sharded?`?

Reply:
Nope ;)